src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -94,8 +94,8 @@ export class AsyncCreatableSelectInput extends React.Component {
               'react-select-warning': this.props.hasWarning,
             })}
             components={{
-              ...this.props.components,
               ...customizedComponents,
+              ...this.props.components,
```

Comment:
Maybe I am just not getting the naming here. Customized are the ones you pass in? `props.components` are the ones the parent component passes as a default? Or the other way round? I guess that's why we swap the spread?

Reply:
Other way around. `customizedComponents` are those which customizied (compared to the defaults used by `react-select`)  by ui-kit, and `props.components` are coming from the user of ui-kit `SelectInput`.